www/gatsby-config.js

```diff
@@ -258,6 +258,14 @@ module.exports = {
     `gatsby-plugin-twitter`,
     `gatsby-plugin-react-helmet`,
     `gatsby-plugin-sitemap`,
+    {
+      resolve: `gatsby-plugin-react-svg`,
+      options: {
+        rule: {
+          include: /assets\/(guidelines|icons|ornaments)\/.*.svg$/,
```

Comment:
was this the intention?
```suggestion
          include: /assets\/(guidelines|icons|ornaments)\/.*\.svg$/,
```

Reply:
fixed in  #25765